Assets/MXR.SDK/Plugins/Android/NativeUtils.java

```diff
@@ -194,4 +194,29 @@ public String getInstalledAdminAppPackageName() {
         return null;
     }
 
+    public int getInstalledAdminAppVersionCode() {
+        PackageManager pm = mContext.getPackageManager();
+        List<PackageInfo> packages = pm.getInstalledPackages(0);
+
+        String adminAppPackageName = getInstalledAdminAppPackageName();
+        if (adminAppPackageName == null) return null;
+
+        for (PackageInfo packageInfo : packages) {
+            if (!packageInfo.packageName.equals(adminAppPackageName)) continue;
+
+            int versionCode;
+            // The default versionCode field is deprecated on versions > P.
```

Comment:
What is meant by `P` ?

Reply:
This is an Android constant for version Pie, which is SDK 28. This is common practice in Android land, which we're in with Java plugins.

Reply:
Cool!  I was wondering was it P for product, SGTM!